src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/attribute.yml

```diff
@@ -7,7 +7,7 @@ admin_attributes_index:
     _legacy_controller: AdminAttributesGroups
     redirectQueryParamsToKeep:
       - 'attributeGroupId'
-    _legacy_link: AdminAttributesGroups:viewattribute_group
+    # _legacy_link: AdminAttributesGroups:viewattribute_group
```

Comment:
Why commenting the legacy links again?

Reply:
I prefer to wait until AttributeController is completely migrated to be able to add _legacy_links. A "@todo: add legacy links after page is fully migrated" was already present in the attribute.yml file

Reply:
Also, the `admin_attributes_edit` route is in error if I add a `_legacy_link`. The `attributeId` attribute is missing in the query params